### PR TITLE
New post page

### PR DESCRIPTION
**Pull request by aysam199**
Added the header, image and caption, body and to footer sections.
The design is as close to figma as possible,  fixes might be needed for now ant the future.
New push:
-Fixed the files structure
-Fixed footer design
-Renamed imageCaption component
-Added difficulty section
-Moved the AddPost components into the AddPost page
-Built the logic on how to collect all the data (still needs to add the ingredient and steps: issue 26)
-Fixed the upload image, now its the component from the wajjabt frontend

rel #9 
rel #15

### CODE REVIEW

File: src/pages/AddPost/index.jsx [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
     return (
         <div>
-            
+            <NewPostHeader/>
+            <ImageCatpion/>
+            <NewPostBody/>
+            <NewPostFooter/>
         </div>
     )
 }
```
Comment (Karyum) [inline src/pages/AddPost/index.jsx]: this is a huge logic flaw, well there is a couple.
1) move all of these components into a new folder called components inside AddPost folder in the pages (so src/pages/AddPost/components)
2) the data is stored on each component individually but let's you want to submit the data to the server, how do you do that ? how do you collect that data from all of these components? you do not have this logic here while it should be here so add it

File: src/components/NewPost/NewPostFooter/index.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+    <div class="foodOptions2">
+      <input type="checkbox" name="check1"/>
+      <label className="foodOptionsName">Kosher</label>
+    </div>
+    <div class="foodOptions3">
+      <input type="checkbox" label="asdsad" name="check2"/>
+      <label className="foodOptionsName">Vegan</label>
+    </div>
+  </div>
```
Comment (Karyum) [inline src/components/NewPost/NewPostFooter/index.jsx]: these look nothing like the design, make them look identical

File: src/components/NewPost/ImageCaption/index.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.css';
+import TextField from '@material-ui/core/TextField';
+
+const ImageCaption = () => (
```
Comment (Karyum) [inline src/components/NewPost/ImageCaption/index.jsx]: this is a bad name for this component, i imagine a component that's ImageCaption to only hold text

File: src/components/NewPost/ImageCaption/index.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  <div>
+    <div className="imageCaption">
+      <i className="fas fa-camera-retro imageIcon"></i>
```
Comment (Karyum) [inline src/components/NewPost/ImageCaption/index.jsx]: didn't you say you were going to add the image upload from wajbat-frontend ?

**Review by Karyum — DISMISSED** [review]
After the seeing the page i can conclude that you didn't design this mobile first in mind, it's very very different from the figma desgin (which it should look identical to it not different)

**Review by okarem — APPROVED** [review]

**Pull request merged**